rspirv/sr/items.rs

```diff
@@ -31,25 +36,120 @@ pub struct Function {
 
 pub struct Module {
     /// The module header.
-    pub header: ModuleHeader,
+    pub header: dr::ModuleHeader,
     /// All OpCapability instructions.
-    pub capabilities: Vec<instructions::Capability>,
+    pub capabilities: Vec<spirv::Capability>,
     /// All OpExtension instructions.
-    pub extensions: Vec<instructions::Extension>,
+    pub extensions: Vec<String>,
```

Comment:
Why have you changed these? I can see pros and cons to both ways round. Also, if this is changed what should happen to the things in instructions atm?

Reply:
Seems sensible, just need to be careful that we keep support of unknown extended instruction sets.